.github/workflows/tests.yaml

```diff
@@ -0,0 +1,331 @@
+# WIP github actions
```

Comment:
still WIP? :)

Reply:
I mean...kind of? Publish package I've deferred deliberately, and publish_image is commented out as well. I'll remove the comment, though.